ext/playbooks/os-gnocchi-install.yml

```diff
@@ -102,5 +126,7 @@
         - "system-crontab-coordination"
   vars:
     galera_address: "{{ internal_lb_vip_address }}"
-    ansible_hostname: "{{ container_name }}"
-    is_metal: "{{ properties.is_metal | default('false') }}"
+    gnocchi_galera_address: "{{ galera_address }}"
+    gnocchi_galera_user: gnocchi
+    gnocchi_galera_database: gnocchi
+    is_metal: "{{ propeties.is_metal|default(false) }}"
```

Comment:
this file is headed towards elimination with the replacement being https://review.openstack.org/#/c/324125/